analytic_tag_dimension/views/analytic_view.xml

```diff
@@ -11,7 +11,7 @@
                         <field name="code"/>
                     </group>
                     <group string="Tags">
-                        <field name="analytic_tag_ids" nolabel="1"/>
+                        <field name="analytic_tag_ids" nolabel="1" readonly="1"/>
```

Comment:
Thanks @pedrobaeza , I was wrong because I was thinking only in case that you already have analytic tags, and you want prevent to create new one on dimension form